source/jormungandr/jormungandr/scenarios/journey_filter.py

```diff
@@ -658,6 +658,25 @@ def apply_final_journey_filters(response_list, instance, request):
     journeys = journey_generator(response_list)
     _filter_too_much_connections(journeys, instance, request)
 
+    origin_mode = get_or_default(request, 'origin_mode', [])
+    filter_non_car_journey = origin_mode == ['car']
+    if filter_non_car_journey:
```

Comment:
Is there a reason to write this on 2 lines and not inline it?
```suggestion
    if origin_mode == ['car']:
```